docs/guides/technical-docs-seo-guide.rst

```diff
@@ -0,0 +1,278 @@
+Technical Documentation Search Engine Optimization (SEO) Guide
+==============================================================
+
+.. meta::
+    :description lang=en:
+        Looking to optimize your Sphinx documentation for search engines?
+        This SEO guide will help your docs be better understood by both people and crawlers
+        as well as help you rank higher in search engine results.
+
+
+This guide will help you optimize your documentation for search engines
+with the goal of increasing traffic to your docs.
+While you optimize your docs to make them more crawler friendly for search engine spiders,
+it's important to keep in mind that your ultimate goal is to make your docs
+more discoverable for your users.
+**You're trying to make sure that when a user types a question into a search engine
+that is answerable by your documentation, that your docs appear in the results.**
+
+This guide isn't meant to be your only resource on SEO,
+and there's a lot of SEO topics not covered here.
+For additional reading, please see the
+:ref:`external resources <guides/technical-docs-seo-guide:External resources>` section.
+
+While many of the topics here apply to all forms of technical documentation,
+this guide will focus on Sphinx, which is the most common
+documentation authoring tool on Read the Docs,
+as well as improvements provided by Read the Docs itself.
```

Comment:
Considering the document has a few sections and subsections, it may be good to have a TOC after this paragraph.

```suggestion
as well as improvements provided by Read the Docs itself.


.. contents:: Table of contents
   :local:
   :backlinks: none
   :depth: 3
```